### DISCUSSION TITLE
SpiralLinux at DistroWatch

### DISCUSSION THREAD

**Discussion by Neocamelus** [SpiralLinux/SpiralLinux-project#123] [General]
Hello, 

I just saw that [SpiralLinux](https://distrowatch.com/table.php?distribution=spiral) now has its own page on DistroWatch.
I hope this will bring more users to this Linux distribution which works really well :thumbsup:

**Comment by geckolinux** [comment]
Hi there,
Very cool, thanks for letting me know! And thanks to all the many users that voted for SpiralLinux when it was still on the DistroWatch waiting list, which undoubtedly helped it get listed faster.

**Comment by carlosEgarc** [comment]
Nice review.

https://distrowatch.com/weekly-mobile.php?issue=20221003#spiral

**Reply by geckolinux** [reply]
Whew, glad it behaved itself on the reviewer's hardware! :relieved:

**Comment by Xispra** [comment]
I just noticed Spiral is in last place (100) on the DistroWatch Page Hit Ranking for the last 6 months with 116 views a day. Ya gotta start somewhere, so at least it is on the DW home page.

**Reply by geckolinux** [reply]
Yep, nice to see that!

**Comment by methanoid** [comment]
I'm a fan of SpiralLinux. Can I suggest we all semi regularly "check in" at https://distrowatch.com/table.php?distribution=spiral   ??

It logs your IP so only one visit a day is counted but if we pushed Spiral up the list it would self sustain as more people would think "oooh I've never heard of that, let's go check it out" and its worth people exploring
